spec/Dynamic-Validator.md

```diff
@@ -59,7 +59,7 @@ The candidate can deposit more than **MIN_DEPOSIT**, but they cannot withdraw it
 ## Delegation
 The stakeholders have the right to choose validators in proportion to their shares.
 This is called delegation, and the stakeholders who have delegated are called delegators.
-The delegation is valid only when the delegatee is eligible and not banned.
+The delegation is valid only when the delegatee is neither eligible nor banned.
```

Comment:
I think we can make this clearer by mentioning 'state'.

"The delegation is valid only when the delegatee is neither in the eligible nor banned state."

Reply:
Looks better. I'll change it.